src/TestSuite/ConsoleIntegrationTestTrait.php

```diff
@@ -37,35 +37,35 @@
      *
      * @var bool
      */
-    private $useCommandRunner = false;
+    protected $_useCommandRunner = false;
```

Comment:
no need to underscore the variable, I think

Reply:
The variables has `_` prefixes in 3.6 though.